config/registry/google/modules/gcp-nodepool.yaml

```diff
@@ -0,0 +1,70 @@
+halt: true
```

Comment:
@juandiegopalomino should this be set to `false`

Reply:
yup